src/app/components/Navbar/index.tsx

```diff
@@ -11,10 +10,10 @@ import Button from "../Button";
 
 export default function Navbar() {
   return (
-    <nav className={`${styles["navbar-wrapper"]} container`}>
+    <div className={`${styles["navbar-wrapper"]} container`}>
       <div className={styles.navbar}>
         <a href="localhost:3000">
-          <Image src="logo.svg" alt="Logo" width={94} height={35} />
+          <img src="logo.svg" alt="Logo" width={94} height={35} />
```

Comment:
Maybe we should get rid of size attributes, so this image will "jump" while loading?

Reply:
Sure, I can try it locally to see how it behaves!